src/zc/buildout/tests/buildout.txt

```diff
@@ -3374,7 +3374,7 @@ or paths to use::
 
     >>> cd(sample_bootstrapped)
     >>> remove('setup.cfg')
-    >>> print_(system([buildout, '-csetup.cfg', 'init', 'demo', 'other', './src']), end='')
+    >>> print_(system([buildout, '-csetup.cfg', 'init', 'demo', 'other', './src']), end='')  # doctest: +ELLIPSIS -DONT_ACCEPT_BLANKLINE
```

Comment:
Can you remove all additions of `# doctest: +ELLIPSIS -DONT_ACCEPT_BLANKLINE`?
When you look at `test_all.py`, the `ELLIPSIS` optionflag is already passed to all test cases, so there is no need to add it.  Yes, I see we already do that a lot, but let's not needlessly add it.

And `-DONT_ACCEPT_BLANKLINE` would set `DONT_ACCEPT_BLANKLINE` to False, but this is already the default in Python, so I don't see why this would be needed.  If I understand the [docs](https://docs.python.org/3/library/doctest.html#doctest.DONT_ACCEPT_BLANKLINE) correctly, this option only needs to be set if you need to check if the output *literally* contains the string `<BLANKLINE>`.